.gitignore

```diff
@@ -19,6 +19,7 @@
 .env.production.local
 .eslintcache
 
+/src/components/auth/poolData.js
```

Comment:
Note: since `poolData.js` was already added to the git history, it will continue to be tracked and show up in future commits.

To stop it from being tracked, you'll probably want to do something like `git rm --cached /src/components/auth/poolData.js`.